perf/perf_UDT_probes.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2019 IBM
+# Author: Naveen kumar T <naveet89@in.ibm.com>
+
+import os
+import platform
+import shutil
+from avocado import Test
+from avocado import main
+from avocado.utils import distro, process
+from avocado.utils.software_manager import SoftwareManager
+
+
+class PerfProbe(Test):
+
+    def setUp(self):
+        '''
+        Install the basic packages to support perf and systemtap-sdt-devel.ppc64le
```

Comment:
is this supported on ppc64le system, if yes, please add check for that as well

Reply:
Its a generic code so not included any specific checks.

Reply:
ah sure maybe than comment need to be rephrased

Reply:
yeah done